core/src/test/java/com/linecorp/armeria/common/logging/RequestContextExporterTest.java

```diff
@@ -165,6 +165,27 @@ void customExportKey() {
                                             "foo", "bar");
     }
 
+    @Test
+    void multipleExporters() throws Exception {
+        final ServiceRequestContext ctx = ServiceRequestContext.of(HttpRequest.of(HttpMethod.GET, "/"));
+        final RequestContextExporter exporter1 =
+                RequestContextExporter.builder()
+                                      .name("exporter1")
+                                      .attr("attrs.attr1", ATTR1)
+                                      .build();
+        final RequestContextExporter exporter2 =
+                RequestContextExporter.builder()
+                                      .name("exporter2")
+                                      .attr("attrs.attr2", ATTR2)
+                                      .build();
```

Comment:
If `exporter1` and `exporter2` have the same name, this test seems to fail.
How about appending a unique number to `AttributeKey`.
```java
private static final AtomicLong counter = new AtomicLong();
stateAttributeKey = AttributeKey.valueOf(RequestContextExporter.class, name + counter.getAndIncrement() + "_STATE");
```